shrielenee.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: Shrie: a quasi summary
+---
+
+## A bit about me
+
+I'm a tech writer, avid reader, journaler, record collector, and peer-pressurer living in :sunny: Austin, TX.
+
+
+## Things I like
+
+I like some things, including but not limited to:
+
+- Flying kites in the sunshine
+- Picking through dusty record bins
+- Cultivating my square foot garden
+- Going on swim-ventures
```

Comment:
I just call that an adventure that involves swimming! In the hill country around Austin there are many swimming holes... swim-ventures are usually a lady friend date 😃